src/notation/view/percussionpanel/percussionpanelpadlistmodel.cpp

```diff
@@ -319,6 +337,39 @@ int PercussionPanelPadListModel::createModelIndexForPitch(int pitch) const
     return modelIndex;
 }
 
+void PercussionPanelPadListModel::applyPadSwapOptions(int fromIndex, int toIndex)
+{
+    muse::UriQuery query("musescore://notation/percussionpanelpadswap");
```

Comment:
It's better to specify explicitly the opening options: sync=true and perhaps modal=true